cli/file/definition_test.go

```diff
@@ -131,10 +133,55 @@ func TestLoadDefinition(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name:          "Should_parse_env_variables",
+			File:          "../testdata/definitions/valid_http_test_definition_with_env_variables.yml",
+			ShouldSucceed: true,
+			EnvVariables: map[string]string{
+				"POKEMON_APP_API_KEY": "my secret key",
+			},
+			ExpectedDefinition: definition.Test{
+				Name:        "POST import pokemon",
+				Description: "Import a pokemon using its ID",
+				Trigger: definition.TestTrigger{
+					Type: "http",
+					HTTPRequest: definition.HttpRequest{
+						URL:    "http://pokemon-demo.tracetest.io/pokemon/import",
+						Method: "POST",
+						Headers: []definition.HTTPHeader{
+							{Key: "Content-Type", Value: "application/json"},
+						},
+						Body: definition.HTTPBody{
+							Type: "raw",
+							Raw:  `{ "id": 52 }`,
+						},
+						Authentication: definition.HTTPAuthentication{
+							Type: "apiKey",
+							ApiKey: definition.HTTPAPIKeyAuth{
+								Key:   "X-Key",
+								Value: "my secret key",
+								In:    "header",
+							},
+						},
+					},
+				},
+			},
+		},
 	}
 
 	for _, testCase := range testCases {
+		envVariables := testCase.EnvVariables
```

Comment:
This is needed otherwise the cleanup function fails.

Reply:
It would be nice if you already added that. I try to always run tests in parallel, because it's faster and helps detecting bugs and race conditions.

I usually do:
```
for _, c := range cases {
  t.Run(c.name, func(t *testing.T){
    cl := c
    t.Parallel()
    // the test
  })
}
```

There are 2 main differences with your code:
1. the copy of the case is done within the `t.Run` func
2. it copies the entire case. Otherwise you can get mixed results

Reply:
You are right. I decided to make it as simple as possible because probably we don't need parallel tests here as it is a very small tool